app/views/wikis/edit.html.erb

```diff
@@ -10,6 +10,18 @@
       <%= f.label :body %>
       <%= f.text_area :body, rows: 10, class: 'form-control', placeholder: "Enter Wiki Body" %>
     </div>
+    <div class="form-group">
+      <% if (current_user.role == 'admin' && @wiki.private == true) || (current_user.role == 'premium' && @wiki.private == true ) %>
+        <%= link_to "Add Yourself As Collaborator", new_wiki_collaborator_path(@user), class: 'btn btn-success' %>
+      <% end %>
```

Comment:
@bjblock - I'm able to get this "add" button to show but when I click on it, I'm getting a `template missing` error. How do I go about fixing that since the create/destroy actions shouldn't have a template? I thought that having a `redirect_to` after the save/destroy action within the controller would eliminate the template issue but it didn't.